packages/mui/src/date-time-range-inputs/date-range-input.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Story } from '@storybook/react';
+import { useState } from 'react';
+import { calendarDecorator } from 'src/@storybook/decorators';
+import { TimeRange } from 'src/date-time-utils';
+import { DateRangeInput } from './range-inputs';
+
+export default {
+  title: 'Date Picker/Date Range Input',
```

Comment:
Date Time / Date Range Input

Reply:
I have changed picker stories to this structure:
- Components
  - Pickers
    - Simple Pickers
      - Date Picker
      - Date Time Picker
      - Time Picker
    - Range Pickers
      - Date Range Picker
      - Date Time Range Picker
      - Time Range Picker 
    - Range Picker Inputs
      - Date Range Picker Input
      - Date Time Range Picker Input
      - Time Range Picker Input

So that stories files was minimized as much as possible.